navbar.php

```diff
@@ -1,36 +1,41 @@
-	<nav class="navbar navbar-expand-lg sticky-top navbar-light navbar-custom">
-		<div class="container-fluid">
-			<div class="logo">
-				<a href="index.php" class="logo">
-					<img src="images/starbulls_logo.png" alt="logo" />
-				</a>
-			</div>
-			<button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
-				<span class="navbar-toggler-icon"></span>
-			</button>
-			<div class="collapse navbar-collapse" id="navbarNavDropdown">
-				<ul class="navbar-nav">
-					<li class="nav-item">
-						<a class="nav-link" href="about.php">ABOUT US</a>
-					</li>
-					<li class="nav-item dropdown">
-						<a class="nav-link dropdown-toggle" href="menu.php" id="navbarDropdownMenuLink" role="button" data-bs-toggle="dropdown" aria-expanded="false">
-							MENU
-						</a>
-						<ul class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
-							<li><a class="dropdown-item" href="#cafe">Cafe</a></li>
-							<li><a class="dropdown-item" href="#breakfastSingles">Breakfast</a></li>
-							<li><a class="dropdown-item" href="#tradWings">Wings</a></li>
-							<li><a class="dropdown-item" href="#dinner">Dinner+Sides</a></li>
-						</ul>
-					</li>
-					<li class="nav-item">
-						<a class="nav-link" href="specialMenu.php">WEEKLY SPECIALS</a>
-					</li>
-					<li class="nav-item">
-						<a class="nav-link" href="apply.php">APPLY NOW</a>
-					</li>
-				</ul>
-			</div>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+<nav class="navbar navbar-expand-lg sticky-top navbar-light navbar-custom">
```

Comment:
nav-active class is not assigned to the navbar

Reply:
so the way this works is that the javascript looks for the page that we are on, once it figures out the page it'll add the nav-active class which then the css will be applied to. It doesn't directly go onto the navbar initially.